scss/application.scss

```diff
@@ -1 +1,14 @@
 @import 'equalizer';
+
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```

Comment:
I recommend separating the variables in different scss files:
`scss/variables/_colors.scss`
`scss/variables/_sizes.scss`
`scss/variables/_fonts.scss`

And import these files in `applications.scss`